app/src/main/java/com/example/iou/bill/activities/DividingItemsActivity.java

```diff
@@ -74,31 +76,63 @@ public void onClick(View v) {
         });
     }
 
-    //TODO: calculate totals for each person
-    // Create a dict: keys are names, values are amounts owed
-    // Get the values from the Bill Item
-    // For loop iterating through items to get each billItem
-
+    // Does the math behind splitting a bill
```

Comment:
This is a hefty algorithm. We'd normally want a clear separation between UI and business logic.

I'd recommend actually bringing this all out into a separate class (e.g. `SplitSettlementCalculator.java`)